com.unity.netcode.gameobjects/Tests/Runtime/BaseMultiInstanceTest.cs

```diff
@@ -53,6 +56,51 @@ public virtual IEnumerator Teardown()
             yield return new WaitUntil(() => Time.frameCount >= nextFrameNumber);
         }
 
+        /// <summary>
+        /// We want to exclude the TestRunner scene on the host-server side so it won't try to tell clients to
+        /// synchronize to this scene when they connect
+        /// </summary>
+        private bool VerifySceneIsValidForClientsToLoad(int sceneIndex, string sceneName, LoadSceneMode loadSceneMode)
+        {
+            // exclude test runner scene
+            if (sceneName.StartsWith(k_FirstPartOfTestRunnerSceneName))
```

Comment:
I'd put at least a debug log somewhere to warn about this. Just in case someone has the brillant idea of trying to use "InitTestScene" as their scene name later.

Reply:
I could... but I am hoping that we will migrate all of this to the MultiProcess stuff once that is all sorted.  If it turns out that this becomes a problem with internal Unity developers creating scenes that start with "InitTestScene", then I will do soemthing like that... but the hope is that this won't happen.  :)